extensions/grpc/runtime/src/main/java/io/quarkus/grpc/runtime/supports/Channels.java

```diff
@@ -255,6 +255,9 @@ public static Channel createChannel(String name, Set<String> perClientIntercepto
                 options.setIdleTimeoutUnit(TimeUnit.MILLISECONDS);
             }
 
+            // Use the convention defined by Quarkus Micrometer Vert.x metrics to create metrics prefixed with grpc.<name>.
+            options.setMetricsName("grpc/" + name);
```

Comment:
Why a slash? Micrometer convention (independent of Vert.x) is always dots.

Perhaps an @See here that points to "extractPrefix" .. if I follow things correctly..

Reply:
It's exactly that. I need a way to pass both the type and the name. I wanted to avoid . As it's the regular convention, / is a lot less used. 

Adding an @see is a great idea, I will add that tomorrow.